tutorial/lesson-3/README.md

```diff
@@ -241,7 +255,12 @@ Run again and verify the delete button is not in the update or edit display.
 The list display include a delete option for each Entry listed. To remove this
 the primary HTML is modified to change the `Actions` label to span two table
 columns instead of three. Then the `html_row` is changed to not include the
-delete icon.
+delete icon. The reference UI implemenation uses a table to display items. The
```

Comment:
Is there an explanation how to change the `reference UI` ?
If there is some document explaining that ,  I would place a link to it.
Otherwise the reader will only know this option exists , but has no idea how to do it.